autobump-maven/src/test/resources/__files/metadata/maven-metadata-corrupt.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Great that there is a test to cover this case. But I would consider making this file a lot smaller. I had to look closely to find the wrong closing tag </latesterrorclosingtag>.
It would be a lot faster for other devs to discover this bug if this file would only contain
<invalid-metadata/> for example.